### PR TITLE
ARTIK Cloud: first version of describing integrating it with resin

### PR DESCRIPTION
**Pull request by imrehg**

### CODE REVIEW

File: pages/integrations/artik.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+If you are planning to connect multiple physical devices to this application, you'll need to repeat these device creation steps for each of them.
+
+## Configuring Resin.io
+
+Go to your [resin.io dashboard](https://dashboard.resin.io) and create a new application with the physical device type you are using (for example a Samsung ARTIK board, or any other). In the tutorial's example a BeagleBone Green Wifi is used (with a temperature sensor module).
+
+In the application dashboard define two application-wide environmental variables for the Device ID and Device Token values. For clarity you can choose `ARTIKCLOUD_DEVICE_ID` `ARTIKCLOUD_DEVICE_TOKEN`, though can use any other value you like. In the application-wise setting just use a placeholder value, such as `REDEFINE`.
+
+![Device type set up](/img/integrations/artik/ARTIKCloud_environment.png)
```
Comment (shaunmulligan) [inline pages/integrations/artik.md]: @imrehg this image seems broken in the preview

File: pages/integrations/artik.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+Later in your `Dockerfile.template` you can then install it as:
+
+```
+COPY package.json ./
+RUN JOBS=MAX npm i --unsafe-perm --production && npm cache clean
+```
+
+Then in your application you can access the environmental variables through `process.env.VARIABLE`, and send messages through the SDK. A very simple example is as follows:
+
+__NOTE__: this code exmample is unfinished
```
Comment (shaunmulligan) [inline pages/integrations/artik.md]: "exmample" -> "example"

Comment (imrehg) [inline pages/integrations/artik.md]: Merged to be able to preview within the team, as this is not a blocking change. Added finished example before production #268 

Also, I generally would not consider WIP sections as release-blocking on our docs, but rather communicate early, communicate often. Might need to create some kind of better signaling (e.g. more fields  besides `__NOTE:__`) of different messages regarding the docs. But so far so good, imho.

File: pages/integrations/artik.md (hunk 1 above)
Comment (imrehg) [inline pages/integrations/artik.md]: Fixed in #266, thanks!

**Pull request merged**